website/snapshots.yml

```diff
@@ -0,0 +1,31 @@
+- name: Home page dark theme
+  url: https://wix.github.io/Detox/
+  execute: document.querySelector('div.toggle_vylO.colorModeToggle_DEke > button').click('data-theme', 'dark')
```

Comment:
AFAIR, the default theme is the dark one. According to the docs, `execute` script is run before capturing the snapshots. So, I assume you'll be storing on Percy service snapshots with the light theme called dark, and vice versa.

Reply:
Yes, I looked over the different themes to ensure dark > light were correctly coded.